standards/aws-tags.md

```diff
@@ -0,0 +1,35 @@
+# Using AWS Tags
+
+[Amazon's documentation](https://aws.amazon.com/answers/account-management/aws-tagging-strategies/) describes tags as:
+
+ >...a simple label consisting of a customer-defined key and an optional value that can make it easier to manage, search for, and filter resources. Although there are no inherent types of tags, they enable customers to categorize resources by purpose, owner, environment, or other criteria.
+
+Using tags helps us identify stakeholders as well as track critical financial metrics
+about our AWS usage. You MUST add Amazon tags to services when they are available.
+
```

Comment:
Ooh ooh, I also made an AWS Config rule on sandbox (and am planning to make it on nypl-dev at some point) that checks for resources that don't have proper tags on them.  You go to the rule, see all the non-compliant resources, and fix them.  
We could make it so new resources made without tags generate an error notification.  It's fairly easy to make it send an email, but I wanted to auto-make a jira ticket.  So far, getting that to work has eluded me.  It might be something with the System Manager permissions.

Reply:
Once upon a time I got Jira tickets to be created by an auto-generated error email.
It's a pain - you **do** need elevated Jira permissions...and you also need to lower the security settings of the email account of the recipient to allow Jira's bot to log in and check emails.

I'm interested in trying out the warnings in the sandbox account to see if there are any down sides. I'd like a tour of the AWS config stuff you've done if/when you get a chance!